cypress/jenkins/cypress.config.jenkins.ts

```diff
@@ -89,6 +90,8 @@ export default defineConfig({
     azureClientSecret:   process.env.AZURE_CLIENT_SECRET,
     customNodeIp:        process.env.CUSTOM_NODE_IP,
     customNodeKey:       process.env.CUSTOM_NODE_KEY,
+    accessibility:       !!process.env.TEST_A11Y, // Are we running accessibility tests?
```

Comment:
In this context, if `accessibility = "false"`, as it's a non-empty string, the `!!`  operation will set the accessibility env var to true. So, I suggest to change it to 
`accessibility: process.env.TEST_A11Y === 'true'`